### PR TITLE
Reworked Shooter Controls

### PR DESCRIPTION
**Pull request by tails618**
Slightly adjusted the way shooting/ending shooting works. May not actually work yet.

### CODE REVIEW

**Comment by tails618** [comment]
@balghane I added that if statement logic to the `initialize()` function in the shooter.

File: src/main/java/frc/robot/command/ShootBalls.java [hunk 1]
```diff
@@ -15,18 +15,23 @@ public class ShootBalls extends Command{
   public static boolean[] shotprogression = new boolean[RobotMap.SHOT_PROGRESSION_LENGTH];
   public static boolean startTimeNeeded = true;
   public static double startTime = 0.0;
-  private boolean shoot;
+  private static boolean shootOn;
```
Comment (balghane) [inline src/main/java/frc/robot/command/ShootBalls.java]: This should not be static I don't think

Comment (tails618) [reply src/main/java/frc/robot/command/ShootBalls.java]: Fixed.

**Review by balghane — COMMENTED** [review]
Looks good, pending testing on the robot

**Comment by tails618** [comment]
Shooter works + commented out most prints + removed intake limits (again)

File: src/main/java/frc/robot/command/ShootBalls.java [hunk 2]
```diff
@@ -15,18 +15,21 @@ public class ShootBalls extends Command{
...
   public static double startTime = 0.0;
-  private boolean shoot;
+  private boolean shootOn;
   public ShootBalls(boolean twoballs, boolean shoot){
     requires(Robot.m_shooter);
     //requires(Robot.m_intake);
     TwoBalls = twoballs;
-    this.shoot = shoot;
+    shootOn = shoot;
+    System.out.println("CONSTRUCTOR");
```
Comment (balghane) [inline src/main/java/frc/robot/command/ShootBalls.java]: Can you remove all these prints now that we don't need them anymore

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 3]
```diff
@@ -122,11 +126,19 @@ public static void disableLimit(){
...
         // } else {
         //     System.out.println("LIMIT ENABLED");
         // }
+=======
+        if(limitDisabled){
+            // System.out.println("LIMIT DISABLED");
+        } else {
+            // System.out.println("LIMIT ENABLED");
+        }
+>>>>>>> shooter-controls
```
Comment (balghane) [inline src/main/java/frc/robot/subsystems/Intake.java]: Fix this conflict

**Review by balghane — CHANGES_REQUESTED** [review]

**Review by balghane — APPROVED** [review]

**Pull request merged**